server/schedule/operator_controller.go

```diff
@@ -306,11 +307,15 @@ func (oc *OperatorController) AddWaitingOperator(ops ...*operator.Operator) int
 		operatorWaitCounter.WithLabelValues(desc, "put").Inc()
 		oc.wopStatus.ops[desc]++
 		added++
+		needPromoted++
```

Comment:
what's difference between added and needPromoted?

Reply:
`needPromoted` is used for the promoting batch. if `isMerge` is true,  `added` will add 2 and `needPromoted` will add 1